### ISSUE TITLE
Remove newlines from qualifier text

### ISSUE THREAD

**Issue by jimrybarski** [jimrybarski/gb-io#1]
When qualifier text (and possibly other things) are long enough to require a newline, that newline is preserved after parsing. For example, take this genbank snippet:

```
/product="Dolichyl-phosphooligosaccharide-protein
          glycotransferase"
```

When `Feature::qualifer_values()` is called, it will result in `Dolichyl-phosphooligosaccharide-protein\nglycotransferase` . This should really be converted back to a space in most cases, though for the `/translation` qualifier, we wouldn't want spaces.

Things to explore/consider: 
1. Can we always known when a newline was added at a whitespace position, and when it was used to divide a very long (spaceless) string?
2. Maybe the best way to do this is to always convert a newline into a single space, and let the user decide if they want to eliminate them? One problem with this is that it eliminates the distinction between spaces that existed in the original string and spaces that replaced newlines. You might have a string that was cut at a non-space boundary but that had spaces elsewhere.
3. Maybe just add an optional flag or alternative function to remove newlines from qualifier text automatically when retrieving them?

**Comment by jimrybarski** [comment]
Another possibility: when adding a newline at a word boundary, retain the space and place it at the beginning of the new line. This way, roundtrip parsing should work without any additional logic.

**Comment by jimrybarski** [comment]
Resolved by ae6c53aa69e4a98e4be17aadc5aaee173a360ace
